meta-lmp-base/recipes-connectivity/bluez5/bluez5_5.65.bbappend

```diff
@@ -0,0 +1,16 @@
+# This enables using BSD library libedit for client/mesh features
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+READLINE_LIB = "${@bb.utils.contains('LMP_DISABLE_GPLV3', '1', 'libedit', 'readline', d)}"
+
+PACKAGECONFIG:append = " client ${READLINE_LIB} "
+
+PACKAGECONFIG[readline] = "--with-readline=readline,,readline,"
+PACKAGECONFIG[libedit] = "--with-readline=libedit,,libedit,"
+PACKAGECONFIG[client] = "--enable-client,--disable-client,${READLINE_LIB}"
```

Comment:
I would say this is also **Upstream-Status: Pending** we might need to think a way to tag recipes that is to be upstreamed in future

Reply:
Thanks, done.